### PR TITLE
Training/2021.10/060 records

### PR DESCRIPTION
**Pull request by leogermond**

### CODE REVIEW

File: courses/fundamentals_of_ada/060_record_types.rst [hunk 1]
```diff
@@ -621,121 +547,119 @@ Quiz
...
-----------------------
-Variant Record Types
-----------------------
+ Explanations
 
-* *Variant record type* is a record type where
+ A. :ada:`C => 100`
+ B. Multiple declaration calls :ada:`Next` twice
+ C. Correct
+ D. :ada:`C => 100` ha no effect on :ada:`A` and :ada:`B`
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/060_record_types.rst:570]: "ha" should be "has"

File: courses/fundamentals_of_ada/060_record_types.rst [hunk 2]
```diff
@@ -621,121 +547,119 @@ Quiz
...
+   + Unless object is **mutable**
 
-      * Constraint_Error raised if the check fails
+------------------------------
+Mutable Discriminated Record
+------------------------------
+
+* When discriminant has a **default value**
+
+   + Any object instanciated **without** a discriminant value is **mutable**
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/060_record_types.rst:626]: instantiated

Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/060_record_types.rst:626]: Not true - if the discriminant has a default, the object is mutable (if you create a derived type and specify the discriminant, then objects of _that_ type are immutable)

File: courses/fundamentals_of_ada/060_record_types.rst [hunk 3]
```diff
@@ -621,121 +547,119 @@ Quiz
...
+  ...
+  S := (Group => Student, Gpa => 0.0);
+  S := (Group => Faculty, Pubs => 10);
 
 -----------
 Semantics
 -----------
 
-* Variable of type `Person` is constrained by value of discriminant supplied at object declaration
+* :ada:`Person` objects are **constrained** by their discriminant
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/060_record_types.rst:648]: This section is NOT TRUE for discriminants with defaults

Comment (leogermond) [reply courses/fundamentals_of_ada/060_record_types.rst:648]: Rewrote it

**Pull request merged**